packages/ringcentral-widgets/components/ActiveCallPanel/index.js

```diff
@@ -161,7 +162,30 @@ class ActiveCallPanel extends React.Component {
         </span>
         )}
       />);
-
+    const isMergeCtrl = layout === callCtrlLayout.mergeCtrl
```

Comment:
is~ means something can be represented as bool, not a component, better renaming it